voice/bxml/callbacks/conferenceCreated.md

```diff
@@ -1,19 +1,28 @@
 {% method %}
 ## Conference Created Event
 The Conference Created event is fired whenever a new [conference](../verbs/conference.md) that specified a callbackUrl is created.
+
+`<PlayAudio>` and `<SpeakSentence>` verbs can be used in BXML returned by this callback to play audio in the conference, which will be heard by members that enter the conference while still playing.
```

Comment:
Let's make it clear that you can have no content at all, or you can have BXML. As it is, it sounds like you might have to return `<Response/>` if you want to do nothing when that isn't the case

Reply:
Reworded a bit, adding "can be either empty or a BXML" to the beginning.